cdap-master/pom.xml

```diff
@@ -259,6 +249,10 @@
       <artifactId>mockito-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
```

Comment:
Why is this dependency required?

Reply:
I don't remember adding this, but as per the blame, you added this dependency and there are several occurrences such as `Locations.java`, `SchemaTest` and 'logback-test.xml`.